engine/rpcserver.go

```diff
@@ -1600,55 +1600,65 @@ func (s *RPCServer) GetAuditEvent(_ context.Context, r *gctrpc.GetAuditEventRequ
 
 // GetHistoricCandles returns historical candles for a given exchange
 func (s *RPCServer) GetHistoricCandles(_ context.Context, req *gctrpc.GetHistoricCandlesRequest) (*gctrpc.GetHistoricCandlesResponse, error) {
-	if req.Exchange == "" {
+	if !req.UseDb && req.Exchange == "" {
```

Comment:
We can probably drop the DB bool?

Reply:
Nevermind, read the logic from the previous discussion.

Reply:
Will drop it was left over from when i was planning on supporting empty exchange name but as i need an exchange for uniq indexing its easier to not support it (also could not find any reason to have empty exchanges)